package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "serverless-wso2-apim",
-  "version": "0.7.0",
+  "version": "0.6.6-alpha.0",
```

Comment:
Downgraded version

Reply:
I cloned repo before the current PR merged.